src/components/edit-profile/index.tsx

```diff
@@ -113,6 +113,14 @@ export class EditProfile extends React.Component<Properties, State> {
     );
   }
 
+  getNoneOption() {
+    return {
+      id: 'None (wallet address)',
```

Comment:
I think some sort of constant is more appropriate for an id. ZID_NONE or something.